src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -558,6 +572,7 @@ def get_request_status(
     verbose: Optional[bool] = False,
     include_identities: Optional[bool] = False,
     download_csv: Optional[bool] = False,
+    sort_due_date: Optional[ColumnSort] = None,
```

Comment:
This could be switched for `sort_field` and `sort_direction`

Reply:
I implemented this using the `ColumnSort` enum that I made. This makes it so fastapi/pydantic validate that only `asc` and `desc` are being used at options for `sort_direction`